### ISSUE TITLE
[Question]: Parsing for git support

### ISSUE THREAD

**Issue by aarondill** [cafkafk/eza#107]
What's the recommended way to test (in a shell) for git support.
```shell
> eza --version
eza - A modern, maintained replacement for ls
v0.10.4 [+git]
https://github.com/cafkafk/eza
> eza --version | head -n2 | tail -n1 | grep -qF '[+git]'
> eza --version | grep -qF '[+git]'
```

Can you be guaranteed the features will always be on the second line?
Will the git feature always be '[+git]' or '[-git]'?
Should I instead search for '+git'?

**Comment by aarondill** [comment]
Some context: I'd like to use this to set up my alias to call `e[xz]a --git` if support is enabled, otherwise just `e[xz]a ... [other opts]`, to avoid the "feature disabled" error
```
exa: Options --git and --git-ignore can't be used because `git` feature was disabled in this build of exa
```

**Comment by cafkafk** [comment]
> Some context: I'd like to use this to set up my alias to call `e[xz]a --git` if support is enabled, otherwise just `e[xz]a ... [other opts]`, to avoid the "feature disabled" error

I cannot promise this will never change, but this may work for your use case:

```shell
eza --version | grep '\[+git\]' -o && eza -l --git || eza -l
```

Can you see if this works for you?

**Comment by cafkafk** [comment]
Of course, you should be able to add any options you like to either of those
